handbook/product/licensing.md

```diff
@@ -3,22 +3,72 @@
 
-As described [in the open source FAQ](../community/faq.md#is-all-of-sourcegraph-open-source), users can use just that functionality without agreeing to any enterprise licensing terms by building their own server image. If they do so, no code from our enterprise licensed features nor any trademarks or logos will be included in their Sourcegraph deployment. This means that there will be no references to functionality such as batch changes.
+### Sourcegraph OSS
+
+As described [in the open source FAQ](../community/faq.md#is-all-of-sourcegraph-open-source), users can use Sourcegrah OSS without agreeing to any enterprise licensing terms by building their own server image. If they do so, no code from our enterprise licensed features nor any trademarks or logos will be included in their Sourcegraph deployment.
+
+In practice, Sourcegraph OSS involves:
+
+- Removing the `enterprise` directories from the repo
+- Building your own docker image (you can’t just use ours)
+
+This means you have the following limitations:
+
+- No path to upgrade to an Enterprise or Team plan down the line (you can’t change your mind without launching a new instance)
+- No external dependencies (means no extensions!)
+- No enterprise features (SSO, access control, Batch Changes, Code Insights, Code Monitoring)
+- Built as a single container (limited scalability)
+- No access to our extensions
+
+Benefits to the OSS option include:
+
+- No user limit
+- Some people prefer OSS ideologically
+
+If someone is talking to Sales or CE and says they’re using the OSS version, they are 99% of the time using the free tier of Enterprise Sourcegraph. If they say they’re hitting a user limit, they’re using the free version of Sourcegraph, not Sourcegraph OSS.
+
+### Sourcegraph Enterprise
 
 Most users, however, will use our prebuilt images, which include code licensed under our [enterprise license](https://github.com/sourcegraph/sourcegraph/blob/main/LICENSE.enterprise), and require agreeing to our [terms of service](https://about.sourcegraph.com/terms/) and [privacy policy](https://about.sourcegraph.com/privacy). Users who use these images — with or without a paid plan — are agreeing to be bound by the terms of the enterprise license in addition to the terms of the Apache License.
 
+If someone follows the install instructions for our Docker, Docker Compose, or Kubernetes options ([reference](https://docs.sourcegraph.com/admin/install)), they will install Sourcegraph Enterprise.
+
+Because the code for all three pricing plans is the same, customers can subscribe to any plan in any of our [deployment methods](https://docs.sourcegraph.com/admin/install) except on Sourcegraph Cloud where packaging and pricing are not defined, and there are no plans yes. The deployment method a customer uses is unrelated to the pricing plan, generally (for exceptions, ping the CE team).
```

Comment:
```suggestion
Because the code for all three pricing plans is the same, customers can subscribe to any plan in any of our [deployment methods](https://docs.sourcegraph.com/admin/install) except on Sourcegraph Cloud, where packaging and pricing are not defined, and there are no plans yet. The deployment method a customer uses is unrelated to the pricing plan, generally (for exceptions, ping the CE team).
```